server/src/port/adapter/service/blockchain/ethereum/web3/test/socket/config/web3.ts

```diff
@@ -1,4 +1,4 @@
 export default {
     url: 'localhost:8545',
-    type: 'socket',
+    type: 'http',
```

Comment:
Should the directory be changed `server/src/port/adapter/service/blockchain/ethereum/web3/test/socket/config/web3.ts` to `server/src/port/adapter/service/blockchain/ethereum/web3/test/http/config/web3.ts`??